pyamplipi/amplipi.py

```diff
@@ -43,7 +43,7 @@ def __init__(
             verify_ssl,
             disable_insecure_warning,
         )
-        self.version: Optional[Tuple[int]] = None
+        self.version: Optional[Tuple[int, ...]] = None
```

Comment:
Tuple[int] means a tuple with a single value (ie, `(0)`). Tuple[int, ...] means a tuple of ints of arbitrary length. I think we use tuples like `(1, 2, 3)` to describe version 1.2.3.